QModManager/OptionsManager.cs

```diff
@@ -31,14 +33,21 @@ internal static void Postfix(uGUI_OptionsPanel __instance)
                 ModsTab = __instance.AddTab("Mods");
                 __instance.AddHeading(ModsTab, "QModManager");
 
-                bool enableDebugLogs = PlayerPrefsExtra.GetBool("QModManager_EnableDebugLogs", false);
-                __instance.AddToggleOption(ModsTab, "Enable debug logs", enableDebugLogs,
-                    new UnityAction<bool>(toggleVal => PlayerPrefsExtra.SetBool("QModManager_EnableDebugLogs", toggleVal)));
+                __instance.AddToggleOption(ModsTab, "Enable debug logs", Utility.Logger.EnableDebugLogging,
+                    new UnityAction<bool>(toggleVal => ChangeDebugLogs(toggleVal)));
 
-                bool updateCheck = PlayerPrefsExtra.GetBool("QModManager_EnableUpdateCheck", true);
-                __instance.AddToggleOption(ModsTab, "Check for updates", updateCheck,
+                __instance.AddToggleOption(ModsTab, "Check for updates", PlayerPrefsExtra.GetBool("QModManager_EnableUpdateCheck", true),
                     new UnityAction<bool>(toggleVal => PlayerPrefsExtra.SetBool("QModManager_EnableUpdateCheck", toggleVal)));
+
+                __instance.AddToggleOption(ModsTab, "Enable developer mode", DevMode, new UnityAction<bool>(toggleVal => DevMode = toggleVal));
             }
         }
+
+        internal static void ChangeDebugLogs(bool value)
+        {
+            string path = Path.Combine(Environment.CurrentDirectory, "QModDebug.txt");
+            if (value && !File.Exists(path)) File.WriteAllText(path, "1");
+            if (!value && File.Exists(path)) File.Delete(path);
```

Comment:
Consider [File.Create](https://docs.microsoft.com/en-us/dotnet/api/system.io.file.create?view=netframework-4.8) instead of writing a "1" to the file.  
It might cause confusion among people expecting to be able to change this to a "0" and have it disable logs.

Also, please,  
add an empty line before any if-statement,
and add keep the executing statements of an if-statement on their own line.

Reply:
This is irrelevant after #125 is merged.

Reply:
Marking this as Unresolved until #125 is _actually_ merged